.github/workflows/smoke_test.yml

```diff
@@ -0,0 +1,60 @@
+name: AWS SDK Smoke Tests
+on:
+  push:
+    branches:
+      - main
+      - stable
+      - next
+    paths:
+      - 'packages/aws_sdk/**/*.dart'
+      - 'packages/smithy/**/*.dart'
+  pull_request:
+    paths:
+      - 'packages/aws_sdk/**/*.dart'
+      - 'packages/smithy/**/*.dart'
+  schedule:
+    - cron: "0 0 * * 0"
+defaults:
+  run:
+    shell: bash
+permissions: read-all
+
+jobs:
+  smoke_test:
+    name: Smoke Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
nit: for these refs do you mind adding a comment saying which version number it is?